sdk/communication/azure-communication-chat/azure/communication/chat/_shared/user_credential.py

```diff
@@ -79,12 +80,14 @@ def _wait_till_inprogress_thread_finish_refreshing(self):
         self._lock.acquire()
 
     def _token_expiring(self):
-        return self._token.expires_on - self._get_utc_now() <\
+        return self._token.expires_on - self._get_utc_now_as_int() <\
             timedelta(minutes=self._ON_DEMAND_REFRESHING_INTERVAL_MINUTES)
```

Comment:
This still wont work - you can do a `<` operation between an int and a timedelta.

Reply:
fixed